lib/Order.rb

```diff
@@ -0,0 +1,3 @@
+class Orders
```

Comment:
As this is a class for a single order, consider changing the class name to 'Order'